chart/customer/templates/server.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    debliberty: "true"
```

Comment:
Used this annotation `bluecompute: "true"` in Catalog & Inventory and defined the same in prometheus charts as well. Can you please use the same here.